Assets/Scripts/MovingElements/EQ Visualization/SampleProcessor.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEngine.Profiling;
+
+public class SampleProcessor : MonoBehaviour
+{
+    [SerializeField]
+    MusicSampler Sampler;
+
+    public int NumBins;
```

Comment:
Should be Property (not public)

Reply:
This needs to be accessed in both the Unity editor and other scripts, making it a property makes the field not appear in the editor, even when serializing it. If you know some way around this, let me know.

Reply:
changed, with slightly more complexity to allow the value to still be changeable in editor